packages/opencensus-nodejs/README.md

```diff
@@ -1,12 +1,109 @@
+tracing.tracer.startRootSpan(rootSpanOptions, (rootSpan) => {
+
+    // You can create as many child spans as needed
+    childSpan = tracing.tracer.startChildSpan(name: 'your child span');
+    // Do some operation...
+    // Finish the child span at the end of it's operation
+    childSpan.end();
+
+    // Finish the root span at the end of the operation
+    rootSpan.end();
+});
+```
+
+### Tracing Options
+
+Tracing has many options available to choose from. At `tracing.start()`, you can set the following:
+
+| Options | Type | Description |
+| ------- | ---- | ----------- |
+| [`bufferSize`](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/config/types.ts#L25) | `number` | Maximum size of a buffer |
+| [`bufferTimeout`](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/config/types.ts#L27) | `number` | Max time for a buffer can wait before being sent |
+| [`logger`](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/config/types.ts#L29) | `Logger` | A logger object |
+| [`samplingRate`](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/config/types.ts#L35) | `number` | Determines the samplin rate. Ranges from 0.0 to 1.0 |
+| [`ignoreUrls`](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/config/types.ts#L37) | `Array<string>` | Determines the ignored (or blacklisted) URLs |
+| [`logger`](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/config/types.ts#L39) | `Logger` | A logger object |
+| [`propagation`](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/config/types.ts#L41) | `Propagation` | A propagation instance |
+| [`logLevel`](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/config/types.ts#L47) | `number` | Level of logger - 0:disable, 1: error, 2: warn, 3: info, 4: debug |
+| [`maximumLabelValueSize`](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/config/types.ts#L52) | `number` | The maximum number of characters reported on a label value |
+| [`plugin`](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/config/types.ts#L68) | `PluginNames` | A list of trace instrumentations plugins to load |
+| [`exporter`](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/config/types.ts#L70) | `Exporter` | An exporter object |
+| [`logger`](https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-core/src/trace/config/types.ts#L72) | `Logger` | An instance of a logger |
+
+**TODO: Choose between the two options**
```

Comment:
If this refers to the table vs. the bulleted list, I prefer the table.

Reply:
Awesome, that was my opinion too haha